policyengine_uk/parameters/gov/dfe/childcare_funding_rate.yaml

```
@@ -0,0 +1,23 @@
+description: The Department for Education converts childcare hours into expenses for childcare entitlement at these rates.
+metadata:
+  type: single_amount
+  threshold_unit: year
+  amount_unit: currency-GBP
+  label: Childcare entitlement hourly rates by age
+  reference:
+    - title: Commons Library
+      href: https://commonslibrary.parliament.uk/research-briefings/cbp-8052/
+
+brackets:
+  - threshold:
+      2015-01-01: 0
+    amount:
+      2015-01-01: 11.22
```

Since this is for 2024/25
```suggestion
      2024-04-01: 11.22
```